### PR TITLE
new resource `azurerm_spring_cloud_certificate`

### PR DESCRIPTION
**Pull request by njuCZ**
image

### CODE REVIEW

File: azurerm/internal/services/appplatform/spring_cloud_certificate_resource.go [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
```
Comment (jackofallops) [inline azurerm/internal/services/appplatform/spring_cloud_certificate_resource.go:43]: No length or character constraints on the service here?

**Review by jackofallops — CHANGES_REQUESTED** [review]
Hi @njuCZ 
Thanks for this new resource, just one really minor grammar change for the docs and a query below, other than that I think this is good to merge. 

Thanks!

**Comment by jackofallops** [comment]
Tests pass:
image

File: azurerm/internal/services/appplatform/spring_cloud_certificate_resource.go (hunk 1 above)
Comment (njuCZ) [reply azurerm/internal/services/appplatform/spring_cloud_certificate_resource.go:43]: In the portal, the `name` is auto generated, users does not need to input it. I could not get any constraints for this field. So for now, I just set it `NotEmpty`.

**Pull request merged**